pkg/resources/grant_privileges_to_share_acceptance_test.go

```diff
@@ -0,0 +1,494 @@
+package resources_test
+
+import (
```

Comment:
nice tests! Can we get at least a few negative ones? (e.g. privilege not matching the object type, no `on` option provided)